src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -434,13 +419,11 @@ void AttitudeEstimatorQ::task_main()
 			_att_pub.publish(att);
 		}
 	}
-
-	orb_unsubscribe(_sensors_sub);
 }
 
 void AttitudeEstimatorQ::update_parameters(bool force)
 {
-	if (_params_sub.updated()) {
+	if (_params_sub.updated() || force) {
```

Comment:
What is the use case for force.  What is the result of the copy if there was no update?

Reply:
I think that's to initialize.

Reply:
Yes to set the first param values.